matching-notification-resources/src/main/resources/resources/uicomponents/matchingNotification/unnotifiedMatchesTable.js

```diff
@@ -123,12 +119,11 @@ var PhenoTips = (function (PhenoTips) {
     _checkScore : function(scoreFieldName, messagesFieldName) {
         var score = this._$('#' + scoreFieldName).val();
         if (score == undefined || score == "") {
-            this._utils.showHint(messagesFieldName, "$services.localization.render('phenotips.matchingNotifications.emptyScore')");
-            return;
+            return 0;
         } else if (isNaN(score)) {
             this._utils.showHint(messagesFieldName, "$services.localization.render('phenotips.matchingNotifications.invalidScore')");
             return;
```

Comment:
I think this should `return undefined` now so that this can be caught in a check above and search cancelled